src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -153,6 +153,17 @@
     form#order amp-selector[name=color] div:not([selected]):not([disabled]) {
       opacity: 0.6;
     }
+    .animated {
```

Comment:
I'd merge this class into the fadeIn class as these parameters are usually specific to an animation.